### ISSUE TITLE
Explain/document the push-to-talk feature in the user interface when a participant mutes or gets muted

### ISSUE THREAD

**Issue by nekohayo** [jitsi/jitsi-meet#8322]
I did not know at all that meet.jit.si had PTT implemented already, until I dug around in bug reports here. And I'm a geek, so imagine people who wouldn't even know spacebar Push-to-Talk is a thing!

So this is simply a discoverability issue.
An easy way to address this, therefore, is to change the popup message that (already) says this when try to speak into a muted microphone:

> "Trying to speak? You are muted. [unmute]"

And replace it by this (or something similar-but-clear):

> "Trying to speak? You are currently muted. You can unmute yourself by clicking the button below, or you can push-to-talk by pressing and holding the space bar on your keyboard, which will temporarily unmute you while the spacebar key is held pressed down."

If you want a shorter version, maybe the popup message could have a link to a secondary popup message with the explanation and screenshots or something. It could say for example:

>  "Trying to speak? You are currently muted. You can unmute yourself by clicking the button below, or you can push-to-talk with the keyboard's space bar  [unmute] [learn more about push-to-talk]"

TLDR: please make the feature _obvious_. It will be very useful to a lot of people!

**Comment by damencho** [comment]
Such long messages are not suitable for notifications.
If you press shift + ? or the button from the menu "View shortcuts", you will get a list of all shortcuts where you can see and the one for PTT.

**Comment by nekohayo** [comment]
Non-technical users will _not_ go looking for documentation / digging into menus. I can 100% guarantee that to you.

Hence my alternative suggestion above. You could make it even shorter (to the extreme) by doing this if you prefer:

> Trying to speak? You are currently muted.
>    [Unmute]  [Learn about push-to-talk]

...and the 2nd button shows them the info overlay with the explanation about how PTT works.

**Comment by MagicFab** [comment]
@nekohayo as an alternative if you self-host you can modify the language file to show a better message. Agreed it would be of great benefit to improve discoverability for this feature.
I've been away from Jitsi for some time but perhaps we could put together a less-than-one-page "Getting started" leaflet to share with first-time users. I know many teachers who would love this.
Rest assured other popular tools (Google Classrooms) have worse usability, at least here we can hope, propose or implement improvements.

**Comment by nekohayo** [comment]
The problem with the one-page-leaflet-intro-tutorial approach is that 1) people tend to not read when they're in a hurry / stressed, and that's often what may happen when they first join  2) they will forget.

That's why I thought it made more sense to have it in the notification, as it would always be there as reminder.
